### DISCUSSION TITLE
Group photosets

### DISCUSSION THREAD

**Discussion by wacher74** [TumblThreeApp/TumblThree#420] [Q&A (Question & Answer)]
Hi guys
Could somebody explain how to group photosets?

I tried filename templates but
%i doesnt work
%n doesnt work
%e insert not datetime of post date but current datetime
Group photo set option doesnt make anything.

%i-%n-%f just makes "-filename.ext"

Thank you.

**Comment by thomas694** [comment]
Which blog type do you try to download?
Are you sure the post is actually a photo (set) post? Normal text posts with html image links aren't working.

The pattern `%i-%n-%f` should at least produce "postid-filename.ext".

**Comment by wacher74** [comment]
Type: Tumblr Liked by
(I'm sure, not all but lot of posts are set)

General settings:   [  ] Group photo sets

Blog details: Filename template: %i-%n-%f
(there is no group settings)

Neiter %i nor %n is filled, and names like "-e9289ffa3e7d66c3c19690f19adc6b635fc07ec0.jpg" have been created. (With one "-", not two)

I don't insist on the %i or %n, ANY uniqe ID per post or simple sequence would be great in order to keep pictures from one post together.

**Reply by thomas694** [reply]
Oh, I already expected that you don't talk about the normal blog type.
Originally only images and videos were downloadable, without additional information. The "likes" downloader was changed few weeks ago, but the "liked/by" downloader not yet.
Therefore some things like e.g. the filename template aren't working completely yet.

**Comment by wacher74** [comment]
This list is my list of my own likes.
Should I use other blog type?

I tried https://www.tumblr.com/likes but most of posts have been missed, just 2-3 of 10 is downloaded.
The https://www.tumblr.com/liked/by/MYTUMBLR format is download almost everything but no "good" filename.

Both of them are liked-by.

Can I now download my liked images somehow with preserved photosets or i should wait for a next release?
Thank you

PS: Could you close the last files? When I want to delete or move those, I have to close the application.

**Comment by borsun123** [comment]
Hay for me the group photosets doesn't work either. 

The blog type is a hidden blog and im logged in. It has 100% photosets with multiple photos but it just downloads them all and doesn't group them. I have no clue how i should set it up otherwise to make it work.

**Reply by thomas694** [reply]
Can you send 1-2 example post links in the feedback dialog and mention "420"?
